lib/govuk_component/tag_helpers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module GovukComponent
+  module TagHelpers
+    def govuk_tag(body: 'nil', color: nil, **tag_options)
```

Comment:
What are your thoughts on using keyword arguments over options hash @jsugarman @kmahern

Reply:
i like keyword arguments but rails tag helpers (and others like [ViewComponent](https://github.com/github/view_component)  don't use them I believe and so I would emulate that as far as possible - from a practical point of view i think keeping views as concise as possible is preferable, and these args are verbose.